src/main/java/hudson/plugins/sshslaves/SSHLauncher.java

```diff
@@ -801,15 +802,15 @@ private void cleanupConnection(TaskListener listener) {
      * return javaPath if specified in the configuration.
      * Finds local Java, and if none exist, install one.
      */
-    protected String resolveJava(SlaveComputer computer, TaskListener listener) throws InterruptedException, IOException2 {
+    protected String resolveJava(SlaveComputer computer, TaskListener listener) throws InterruptedException, IOException {
```

Comment:
Also I see no usages in private-source plugins, to which I have access.
But maybe you want to create a new method and use the original one as a wrapper. NIT anyway